src/components/Autocomplete/components/ComboBox/ComboBox.tsx

```diff
@@ -266,13 +266,19 @@ export default class ComboBox extends React.PureComponent<Props, State> {
         tabIndex={0}
       >
         <KeypressListener
-          keyCode={Key.DownArrow}
```

Comment:
This would be considered a breaking change and would need to be in a major release

Reply:
I thought `<KeypressListener />` was not exposed and only used internally, so I didn't think it would be breaking, but yes, you're right, it's used in web.